builtin/credential/aws/path_image.go

```diff
@@ -42,6 +42,12 @@ func pathImage(b *backend) *framework.Path {
 				Default:     false,
 				Description: "If set, allows migration of the underlying instance where the client resides. This keys off of pendingTime in the metadata document, so essentially, this disables the client nonce check whenever the instance is migrated to a new host and pendingTime is newer than the previously-remembered time. Use with caution.",
 			},
+
+			"disallow_reauthentication": &framework.FieldSchema{
+				Type:        framework.TypeBool,
+				Default:     false,
+				Description: "If set, allows a client to login only once.",
```

Comment:
might be good to say "If set, only allows a single token to be granted per instance ID. This can be cleared with the auth/aws/whitelist/identity endpoint." Just gives more context as this is in some ways a confusing option.